concepts/string.md

```diff
@@ -0,0 +1,36 @@
+---
+description: String
+---
+
+# String
+
+A string is an object in memory that stores a sequence of characters and colloquially it is the representation of text in the computer.
```

Comment:
I'm thinking something more like this:

> A string is a sequence of characters.  In computer programming, strings are used to represent text.

Reply:
Updated to your suggestion.

Reply:
Do you think it is perhaps worthy to compare the meaning of the term "string" to that of "segment"?
For our translator audience, it's a clear concept that we come across daily.

@zouharvi @bittlingmayer

Reply:
Probably you should add that, not Vilem, since we're not totally sure what translation industry people mean by a segment.